docs/tools/vega-wallet/concepts.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Wallet Concepts
+hide_title: false
+---
+Vega Wallet is an app that lets you manage wallets and cryptographic key pairs. Key pairs are used to sign and send transactions in a secure way. 
+
+Read on to understand the basic concepts of wallets and keys, and how they work. 
+
+## Cryptographic keys
+Each cryptographic key pair includes a public key and a private key. 
+
+Those keys are associated with an entity that needs to authenticate their identity, or sign and send transaction data. The signature is generated through combining the signer's private key with the data being sent in the transaction. Data that is signed with the private key can only be verified with its corresponding public key.
+
+Signing a transaction proves the signer is the owner of the collateral used in the transaction. The network's validator nodes check and authenticate transactions automatically as they come in.  
+
+**Public key**: A public key can be published or shared. 
+
+**Private key**: A private is saved in your wallet file, but is never displayed and is kept secret. 
+
+## Wallet 
+Key pairs are stored within an encrypted and passphrase-protected wallet file so you don't have to manage key pairs yourself, or risk exposing your private key. 
+
```

Comment:
It is a bit redundant with the sentence below:

> A wallet file is encrypted and the information within it can only be accessed with a passphrase.

Both sentence contain good point:
* don't have to manage key pairs yourself, or risk exposing your private key
* file is encrypted
* can only be accessed with a passphrase

I think we should merge them